scripts/setup_sc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Where this script is used?

Reply:
This is used in the Edge test that uses SauceLabs but I had forgotten to add it to the travis config. it is added now in the latest passing version.

Reply:
Great!